packages/block-library/src/video/tracks-editor.js

```diff
@@ -147,8 +146,7 @@ function SingleTrackEditor( { track, onChange, onClose, onRemove } ) {
 					/>
 					<HStack className="block-library-video-tracks-editor__single-track-editor-buttons-container">
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
This can be seen, when you add core/video and add some track list and open in edit mode.

| Before | After     |
| :-------- | :------- |
| <img width="404" alt="Close-track-before" src="https://github.com/user-attachments/assets/e893c1da-5a56-4f65-925f-f3c305abb804"> | <img width="404" alt="close-track-after" src="https://github.com/user-attachments/assets/08953e08-f454-4fb1-8676-56731ae51550"> |